refactor: 코드 정리

chaeyoungeee opened the pull request.

chaeyoungeee committed:
refactor: 코드 정리

chaeyoungeee committed:
refactor: 코드 정리

SeolLab committed:
Merge branch 'develop' into feat-color_technology_cards

SeolLab committed:
컴퓨터 플레이어 화면에 덱추가

SeolLab committed:
Merge branch 'feat-color_technology_cards' of https://github.com/SEdreamers/Do_yoUNO into feat-color_technology_cards

SeolLab committed:
Delete game.py

SeolLab committed:
Create game.py

잘못 지웠다가 다시 업로드 합니다;;

SeolLab committed:
컴퓨터 플레이어 게임 가능.

추후 버그 찾아보고 더 정교하게 만든 담에 재업로드 할게;

SeolLab committed:
Delete card.py

SeolLab committed:
setting <-> main 간 버그 수정

1) setting.py에서 import main 하지 않고(circular import error 방지하고자), Setting 클래스의 gets()함수 사용하고, go back 버튼 누르면 run()함수를 return 통해 탈출해 main으로 돌아오게 만들 경우:  setting에서 사이즈 설정하고 main 넘어 올 때, 화면깨지는 오류발생.

2) import main을 setting.py에 넣음으로써 circular import error에 대한 위험성이 생겼지만, setting에서 사이즈 설정하고 main 넘어 올 때, 화면깨지는 오류는 발생하지 않음. 다만 이 경우에 main.main(window_size[0], window_size[1],self.color_blind_mode)로 전달해 main()이 다시 수행될 때, color_blind_mode 인자를 함께 전달해 수행하도록 함.

SeolLab committed:
Merge branch 'feat-color_technology_cards' of https://github.com/SEdreamers/Do_yoUNO into feat-color_technology_cards

chaeyoungeee committed:
feat: 플레이 방향 표시

SeolLab reviewed (commented):
오 확인해쓰.. 이런게 있으면 확실히 좋을 듯 👍
근데 상대 턴일때 나 가리킬 때가 있는데 수정해야 될 거 같아

shso127 committed:
refacoor : 카드 사이즈 조정, 게임 종료되게 변경

게임 화면을 줄이거나 늘릴 때 카드만 안 줄어든 부분에서 클래스 부분에 screen을 더 둬서 잘 바뀌도록 조절했어. 그리고 single play 할 때 종료를 눌러도 종료가 안되는 것도 그냥 True를 넣어서 안되더라고, 그것도 self.running 넣어서 해결했으

shso127 committed:
Merge branch 'feat-color_technology_cards' of https://github.com/SEdreamers/Do_yoUNO into feat-color_technology_cards

chaeyoungeee commented:
오 맞네 다음에 수정할 때 수정할겡!

SeolLab commented:
![](https://github.trello.services/images/mini-trello-icon.png) [컴퓨터 플레이어](https://trello.com/c/9jQnjUFX/16-%EC%BB%B4%ED%93%A8%ED%84%B0-%ED%94%8C%EB%A0%88%EC%9D%B4%EC%96%B4)
